docs/core/resilience/index.md

```diff
@@ -0,0 +1,152 @@
+- Configures the pipeline with a retry strategy, circuit breaker strategy, and timeout strategy.
+
+Each pipeline is configured for a given `key`, and each `key` is used to identify it's corresponding `ResiliencePipeline` when getting the pipeline from the provider. The `key` is a generic type parameter of the `AddResiliencePipeline` method.
+
+### Resilience pipeline builder extensions
+
+To add a strategy to the pipeline, call any of the available `Add*` extension methods on the `ResiliencePipelineBuilder` instance.
+
+- `AddRetry`: Try again if something fails, which is useful when the problem is temporary and might go away.
+- `AddCircuitBreaker`: Stop trying if something is broken or busy, which benefits you by avoiding wasting time and making things worse.
+- `AddTimeout`: Give up if something takes too long, which can improve performance by freeing up resources.
+- `AddRateLimiter`: Limit how many requests you make or accept, which enables you to control load.
+- `AddFallback`: Do something else when experiencing failures, which improves user experience.
+- `AddHedging`: Do more than one thing at the same time and take the fastest one, which can improve responsiveness.
+
+For more information, see [Resilience strategies](https://www.pollydocs.org/strategies/index.html).
+
+## Enrichment
+
+Enrichment is the automatic augmentation of telemetry with well-known state, in the form of name/value pairs. For example, an app might emit a log that includes the _operation_ and _result code_ as columns to represent the outcome of some operation. In this situation and depending on peripheral context, enrichment adds _cluster name_, _process name_, _region_, _tenant id_ and more to the log as it's sent to the telemetry backend. Enrichment is done systematically and behind the scenes relative to logging and metrics update within the app's main code.
+
+Consider 1,000 instances of a service, spread around the world, each pumping logs and metrics out. If you look at a dashboard for your service and there's a problem, you need to know which region/data center is having trouble. Enrichment automatically makes sure that the log records have the state needed to pinpoint failures in distributed systems. If this isn't done through enrichment, it means the app code itself would need to track this state internally, weave it to wherever logging is done in the code, and then manually emit it. Enrichment makes that simpler, and invisible to app logic.
+
+### Add resilience enrichment
+
+In addition to registering a resilience pipeline, you can also register resilience enrichment. To add enrichment, call the `AddResilienceEnrichment` extensions method on the `IServiceCollection` instance.
+
+```csharp
+services.AddResilienceEnrichment();
+```
+
+By calling the `AddResilienceEnrichment` extension method, you're adding an additional dimensions on top of the default ones that are built-in to the underlying Polly library. The following enrichment dimensions are added:
```

Comment:
Might be good to mention default metrics alongside the tags:
https://www.pollydocs.org/advanced/telemetry.html#metrics

And then what additional tags are added.